src/winetricks

```diff
@@ -1028,58 +1044,73 @@ winetricks_selfupdate_rollback()
 # Caches downloads in winetrickscache/$packagename
 w_download_to()
 {
+		# Export downloader and/or torify
     winetricks_download_setup
 
     _W_packagename="$1" # or path to download file to
-    _W_url="$2"
-    _W_sum="$3"
-    _W_file="$4"
-    _W_cookiejar="$5"
+    _W_url="$2" # Any url
+    _W_sum="$3" # Any shasum hash
+    _W_file="$4" # Any file
```

Comment:
Not any file, it's what the downloaded file should be named.

Reply:
resolved